internal/server/v1/info/golden/variable_info/total_crimes.json

```diff
@@ -99,11 +99,11 @@
             "seriesKey": {
               "observationPeriod": "P1Y"
             },
-            "earliestDate": "2008",
+            "earliestDate": "2020",
```

Comment:
@manishvats2 @Bipnabraham 
Please check on why FBIGovCrime is showing earliest data as 2020 instead of 2008 (https://screenshot.googleplex.com/6NrjZaHZRnjsbN2)